<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enregistreclients</class>
 <widget class="QWidget" name="enregistreclients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>enregistreclient</string>
  </property>
  <widget class="QTextBrowser" name="infoclient">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>23</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelnomclient">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>24</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SAIPrenomduClient">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>54</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labeltel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>87</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Téléphone</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelprenomclient">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>53</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SAINomduClient">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>24</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNValider">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNouveau">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SAITelephone">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>87</y>
     <width>340</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewClient">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>621</width>
     <height>211</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
